main.cpp

```diff
@@ -394,37 +456,43 @@ void Garage::repair(Truck vehcile)
 int main()
 {
     // 1.
-    Dishwasher dishwasher (1500);
-    dishwasher.washDishes(250);
-    dishwasher.lockDoor();
-    dishwasher.informUser();
-    dishwasher.doesSomethingInteresting();
-    std::cout << "**** inputVoltage : " << dishwasher.inputVoltage << std::endl;
-    dishwasher.printinputVoltage();
+    Dishwasher* dishwasher = new Dishwasher (1500);
+    DishwasherWrapper dishwasherWrapper (dishwasher);
+    dishwasher->washDishes(250);
+    dishwasher->lockDoor();
+    dishwasher->informUser();
+    dishwasher->doesSomethingInteresting();
+    std::cout << "**** inputVoltage : " << dishwasher->inputVoltage << std::endl;
+    dishwasher->printinputVoltage();
```

Comment:
This does not demonstrate using your wrapper class
review the example and revise this
Also, create your instance directly in the wrapper's constructor argument list.
Don't create it as a separate variable.
- [ ] revise